client/me/purchases/payment-methods/payment-method-backup-toggle.tsx

```diff
@@ -78,7 +78,7 @@ export default function PaymentMethodBackupToggle( { card }: { card: StoredPayme
 				checked={ isBackup }
 				onChange={ toggleIsBackup }
 				label={
-					translate( 'Use as backup.{{supportLink /}}', {
+					translate( 'Use as backup{{supportLink /}}', {
```

Comment:
FYI this will trigger a new translation, but I think there's a way to convince our i18n tools to treat them as the same? Please check with the i18n team before merging or do a String Freeze.

Reply:
Looks like we can just add the string freeze label and deploy after adding it - p1743455933626989/1743455657.906669-slack-C02AED43D

> The change will trigger a fuzzy match, which should be fine as the placeholder tag was not modified.